test/integration_tests/integration.dart

```diff
@@ -88,12 +95,18 @@ class BrowserManager {
   Future<BrowserTabInstance> createNewTab() async {
     final String targetId = await this.tab.createNewTarget();
 
+    await delay();
+
     final ChromeTab tab =
         await chromeProcess.connectToTabId('localhost', targetId);
-    await tab.connect();
+    await tab.connect(verbose: true);
+
+    await delay();
 
     await tab.wipConnection.target.activateTarget(targetId);
 
+    await delay();
```

Comment:
This turns out to be the important delay; I think chrome wasn't quite ready for the url to change here, and for it to load new page contents.

Reply:
👍, I'll create an issue to track this.